response_operations_ui/views/collection_exercise.py

```diff
@@ -33,8 +33,7 @@ def get_error_message(error_key):
 
 @collection_exercise_bp.route('/<short_name>/<period>', methods=['GET'])
 @login_required
-def view_collection_exercise(short_name, period, error=None, success_message=None, error_message=None,
-                             success_panel=None, show_msg=None):
+def view_collection_exercise(short_name, period):
```

Comment:
One last place still using the previous version of this function: [here](https://github.com/ONSdigital/response-operations-ui/blob/23ba1e407180c8045a0809441f3b6e74293cad82/response_operations_ui/views/collection_exercise.py#L238).

Reply:
The changes to use a redirect are still using `error`, `success_message`, or `error_message` in some/most instances.

Reply:
Will also need to update tests to follow redirects.